src/slurm_plugin/slurm_resources.py

```diff
@@ -31,8 +31,11 @@
     "instance-retirement",
     "instance-stop",
 ]
-
-
+EC2_ICE_ERROR_CODES = {
```

Comment:
nit: you can have this under SlurmNode class since only used there

Reply:
Moved